src/components/Tooltip/component.tsx

```diff
@@ -1,7 +1,108 @@
-import React from "react";
+import {useState, useRef, useEffect, ReactNode} from "react";
+import {motion, AnimatePresence} from "framer-motion"
 
-const Tooltip = () => {
```

Comment:
is this copy pasted / the same as stories/Tooltip? We should consolidate these files and just have one component